arrow-libs/fx/arrow-fx-coroutines/src/commonMain/kotlin/arrow/fx/coroutines/Resource.kt

```diff
@@ -466,7 +465,6 @@ public fun <A> Resource<A>.asFlow(): Flow<A> =
  * instead the caller is responsible for correctly invoking the `release` handler at the appropriate time.
  * This API is useful for building inter-op APIs between [Resource] and non-suspending code, such as Java libraries.
  */
-@DelicateCoroutinesApi
```

Comment:
why was this deleted?

Reply:
It's no longer needed. Or was that added by the authors of the file?

Reply:
Yes, AFAIK we kept it as a safe-guard for user awareness over the ramifications, but if we don't need that anymore, we might want to mention it in the PR.
cc @nomisRev

Reply:
I just put it back. Thanks for the info!